src/lib/isCreditCard.js

```diff
@@ -1,33 +1,34 @@
 import assertString from './util/assertString';
+import isLuhnValid from './isLuhnValid';
 
+const cards = {
+  amex: /^3[47][0-9]{13}$/,
+  dinersclub: /^3(?:0[0-5]|[68][0-9])[0-9]{11}$/,
+  discover: /^6(?:011|5[0-9][0-9])[0-9]{12,15}$/,
+  jcb: /^(?:2131|1800|35\d{3})\d{11}$/,
+  mastercard: /^5[1-5][0-9]{2}|(222[1-9]|22[3-9][0-9]|2[3-6][0-9]{2}|27[01][0-9]|2720)[0-9]{12}$/, // /^[25][1-7][0-9]{14}$/;
```

Comment:
Not sure why this alternative RegExp is in the comments, but maybe we can look at that in #2117

Reply:
it looks like the comment is a more "generalized" version of the final regexp used here.
I would tend to vote to remove the comment, it only causes confusion, but again, this can be part of my refactor #2117

Reply:
Yes, I think removal is the best and also best to do in the refactor